scripts/add_electricity.py

```diff
@@ -214,9 +219,11 @@ def attach_load(n):
     substation_lv_i = n.buses.index[n.buses['substation_lv']]
     regions = (gpd.read_file(snakemake.input.regions).set_index('name')
                .reindex(substation_lv_i))
-    opsd_load = (timeseries_opsd(slice(*n.snapshots[[0,-1]].year.astype(str)),
-                                 snakemake.input.opsd_load) *
-                 snakemake.config.get('load', {}).get('scaling_factor', 1.0))
+    opsd_load = load_opsd_loaddata(load_fn=snakemake.input.load, countries=snakemake.config['countries'])
```

Comment:
```suggestion
    opsd_load = (pd.read_csv(snakemake.input.load, index_col=0, parse_dates=True)
                .filter(items=snakemake.config['countries']))
```
I think it's fine to take a shortcut here without defining the function above.

Reply:
That's a good idea. So, we can save a few lines of code.